src/mame/nec/apc.cpp

```diff
@@ -972,7 +972,7 @@ void apc_state::apc(machine_config &config)
 
 	/* sound hardware */
 	SPEAKER(config, m_speaker).front_center();
-	UPD1771C(config, m_sound, MAIN_CLOCK).add_route(ALL_OUTPUTS, "mono", 1.00); //uPD1771C-006
+	UPD1771C_HLE(config, m_sound, MAIN_CLOCK).add_route(ALL_OUTPUTS, "mono", 1.00); //uPD1771C-006
```

Comment:
Given this is the only other use of the µPD1771C in MAME, is it possible to migrate it and avoid having the extra implementation?

Reply:
Updated the app driver to use the new cpu core and removed the old implementation.